public/models/OnOffTrap.js

```diff
@@ -0,0 +1,48 @@
+if (typeof exports !== 'undefined' && typeof module !== 'undefined' && module.exports) {
+    // used on the server
+    // eslint-disable-next-line global-require
+    Trap = require('./Trap').Trap;
+  }
+
+class OnOffTrap extends Trap {
+    constructor(x, y, width, height, img, value, activeTime, isActive) {
+        super(x, y, width, height, img, value);
+        this.value = value;
+        this.values = [0.001, 0.002, 0.05, 0.005];
+        this.direction = isActive;
```

Comment:
We definitely need a different name for this and refactor it

Reply:
yes we talked about alreeady